src/Raven.Server/Documents/ETL/Providers/Queue/Handlers/Processors/QueueEtlHandlerProcessorForTestAzureQueueStorageConnection.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Storage.Queues;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Operations.ETL.Queue;
+using Raven.Server.Documents.Handlers.Processors;
+using Raven.Server.Web.System;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents.ETL.Providers.Queue.Handlers.Processors;
+
+internal sealed class
+    QueueEtlHandlerProcessorForTestAzureQueueStorageConnection<TRequestHandler, TOperationContext> :
+    AbstractDatabaseHandlerProcessor<TRequestHandler, TOperationContext>
+    where TOperationContext : JsonOperationContext
+    where TRequestHandler : AbstractDatabaseRequestHandler<TOperationContext>
+{
+    public QueueEtlHandlerProcessorForTestAzureQueueStorageConnection([NotNull] TRequestHandler requestHandler) :
+        base(requestHandler)
+    {
+    }
+
+    public override async ValueTask ExecuteAsync()
+    {
+        try
+        {
+            string authenticationJson = await new StreamReader(HttpContext.Request.Body).ReadToEndAsync();
+            Authentication authentication = JsonConvert.DeserializeObject<Authentication>(authenticationJson);
```

Comment:
Why just `Authentication` instead of entire connection string?

Reply:
done